Application/EdFi.Ods.SandboxAdmin/Views/Client/Index.cshtml

```diff
@@ -107,6 +113,12 @@
                             <input id="txtName" type="text" data-bind="value: applicationName, valueUpdate: 'keyup'" class="form-control" placeholder="Name of Sandbox" />
                         </div>
                     </div>
+                    <div class="control-group">
+                        <label class="control-label">Application :</label>
+                        <div class="controls">
+                            <select data-bind="options: applicationList, value:selectedApplication, optionsText: 'Name', valueUpdate: 'keyup' " class="dropdown"></select>
```

Comment:
Since Sandbox Admin uses Bootstrap v4.6 can we use it to style these new dropdowns? [see their official docs](https://getbootstrap.com/docs/4.6/components/dropdowns/).

It would result in something similar to what we already show in the `Manage Accounts` dropdown:

![image](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/assets/158314644/a050b8fb-ff3c-4ea9-b3de-fbdfc414c251)

Reply:
Will be fixed in a follow up story.